app/templates/components/paper-chips.hbs

```diff
@@ -0,0 +1,29 @@
+<md-chips-wrap class="md-chips {{if isFocused 'md-focused'}}">
+  {{#each content as |item|}}
+    <md-chip class="md-chip md-default-theme">
+      <div class="md-chip-content" tabindex="-1" aria-hidden="true">
+        {{#if hasBlock}}
+          {{yield item}}
+        {{else}}
+          <span>{{item}}</span>
+        {{/if}}
+      </div>
+      <div class="md-chip-remove-container">
+        {{#unless readOnly}}
+          <button class="md-chip-remove" {{action 'removeItem' item}} type="button" aria-hidden="true" tabindex="-1">
+            {{paper-icon icon="clear"}}
```

Comment:
This diverges from the angular implementation, which uses its own svg icon called `md-close`. Clear looks similar, but is too big and the positioning is off (perhaps because it is too bid). I'm not sure what the right thing to do here is. If we have to bit off angular's `iconService.js` for other components, such as tags, maybe that's the thing to do? Oh @miguelcobain?